ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -70,7 +71,7 @@ def __init__(self, tracer=None, config=None, service=None):
         super(CIVisibility, self).__init__()
 
         self.tracer = tracer or ddtrace.tracer
-        self._app_key = os.getenv("DD_APP_KEY")
+        self._app_key = os.getenv("DD_APPLICATION_KEY")
```

Comment:
nitpick: 

If it's easy, I think we should be compatible with these three names for application keys: 
`DD_APP_KEY`, `DD_APPLICATION_KEY` and `DATADOG_APPLICATION_KEY`